addons/stock_dropshipping/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def update_stock_picking_type_sequence_code_dropship(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE stock_picking_type
+        SET sequence_code = 'DS'
```

Comment:
You should use the existing sequence code instead of hardcoding it, and check another way of finding dropshipping picking types than the name, as you may rename it.

Reply:
I don't know how to find them in another way.